.mvn/maven.config

```
@@ -1,3 +1,3 @@
 -Pconsume-incrementals
 -Pmight-produce-incrementals
--Dchangelist.format=%d.v%s
+-Dchangelist.format=2.%d.v%s
```

Standard

```suggestion
-Dchangelist.format=%d.v%s
```

suffices IMO.